README.md

```diff
@@ -101,6 +101,16 @@ Please note: implementations may specify authorization parameters for oauth2 or
 zcaps, but not both. Implementations may also not specify any authorization
 parameters, in which case they do not specify `oauth2` or `zcap` properties.
 
+Please check specific test suite READMEs for further details on implementation properties and endpoints. 
+Test suites MAY specify additional properties and features for endpoints such as
+mandatory JSON-LD contexts, keyTypes settings, additional tags for specific tests such as Enveloped Proofs,
+and supported VC Data Model versions. Endpoints in most cases should: be VC-API capable; be able to sign and/or 
+verify using `did:key`; and be able to resolve the following contexts:
```

Comment:
```suggestion
and supported VC Data Model versions. Endpoints in most cases should: be test suite API
capable; be able to sign and/or verify using `did:key`; and be support the following contexts:
```

Reply:
Clarifying the use of "test suite API" and using "support" for contexts rather than "resolve" as implementations should be caching those contexts based on the specs anyhow (and not constantly resolve them).

Reply:
Starts with @BigBlueHat's suggestion, and improves language from there.
```suggestion
and supported VC Data Model versions. In most cases, endpoints are expected to: be test suite API
capable; be able to sign and/or verify using `did:key`; and support the following contexts:
```

Reply:
@BigBlueHat I think we should take @TallTed 's suggestion and get this PR in. Opinions?

Reply:
@aljones15 yes. Please add it.